scalafix-core/shared/src/main/scala/scalafix/internal/util/CombinedSymbolTable.scala

```diff
@@ -0,0 +1,37 @@
+package scalafix.internal.util
+
+import org.langmeta.semanticdb.Symbol
+import scala.meta.Denotation
+import scalafix.SemanticdbIndex
+import org.langmeta.internal.semanticdb._
+import scala.meta.internal.{semanticdb3 => s}
+
+/**
+  * Combination of a SymbolTable and SemanticdbIndex.
+  *
+  * This utility is necessary because SymbolTable contains only global symbols and
+  * SemanticdbIndex has local symbols.
+  */
+class CombinedSymbolTable(index: SemanticdbIndex, table: SymbolTable) {
```

Comment:
Why do we need to pass both `index` and `table` here, given that `EagerInMemorySemanticdbIndex` already takes a symbol table?

Reply:
Good catch, I've merged this into `EagerSemanticdbIndex` avoiding the need for `CombinedSymbolTable`.